src/main/java/com/inqoo/trainingservice/app/exception/ApiExceptionDetails.java

```diff
@@ -0,0 +1,29 @@
+package com.inqoo.trainingservice.app.exception;
+
+import java.time.LocalDateTime;
+
+public class ApiExceptionDetails {
+    private LocalDateTime timestamp;
+    private String message;
+
+    public ApiExceptionDetails(LocalDateTime timestamp, String message) {
```

Comment:
does it have to be public? Make this class immutable

Reply:
done